backend/src/api/interview.js

```diff
@@ -0,0 +1,129 @@
+const express = require('express')
+const router = express.Router()
+const { errorWrap } = require('../middleware')
+const { Interview } = require('../models')
+var mongodb = require('mongodb')
+
+router.get(
+  '/verified-interviewer',
+  errorWrap(async (req, res) => {
+    // const foundKey = await Interview.findById(req.params.interviewerKey);
+    // res.json({ result: foundKey })
+    /*** NOT IMPLEMENTED */
+  })
+)
+
+router.get(
+  '/',
+  errorWrap(async (req, res) => {
+    let interviews
+    interviews = await Interview.find()
+    res.json({ result: interviews })
+  })
+)
+
+router.get(
+  '/:interview_id',
+  errorWrap(async (req, res) => {
+    const retInterview = await Interview.findById(req.params.interview_id)
+    res.json({ interview: retInterview })
+  })
+)
+
+router.post(
+  '/add-interview',
```

Comment:
this can just be a post request with `/interview/`. so it'll look like `router.post('/'` here

Reply:
also once, keys and auth are done, we should also store the the key that submitted this post request into the Interview model.